dcp-0007/dcp-0007.mediawiki

```diff
@@ -0,0 +1,193 @@
+Status: Active
+Created: 2021-06-23
+License: CC0-1.0
+License-Code: ISC
+Requires: DCP0006
+</pre>
+
+==Abstract==
+
+This proposes reverting the currently active maximum expenditure policy of the
+treasury account to the one specified in the original Politeia proposal for the
+decentralized treasury.
+
+==Motivation==
+
+The current algorithm that calculates the maximum expenditure allowed by a set
+of treasury spend transactions within a given window of blocks takes into
+account the average spent in a preceding window.
+
+This is different then the policy that was originally proposed and approved by
+stakeholders in the original Politeia proposal for development of the
+decentralized treasury<ref>[[https://proposals-archive.decred.org/proposals/c96290a]]</ref>
+and has caused an issue that has prevented payments from the treasury account
+from being performed.
+
+This proposal modifies the consensus rules to ensure the maximum expenditure
+policy is in line with what was explicitly approved via Politeia voting, by
+using the total income into the treasury as the reference value to determine the
+spending limit.
+
+The maximum expenditure amount is currently calculated by the <code>dcrd</code> reference
+node implementation by the
+<code>maxTreasuryExpenditure()</code><ref>[[https://github.com/decred/dcrd/blob/afff2fdbcd4c57ade4f0d13e78ad2d3efaebcdec/blockchain/treasury.go#L650]]</ref>
+function. This function defines the following policy:
+
+<blockquote>The sum of treasury spend amounts inside the most recent expenditure window
+cannot exceed the average of the spent amount in the preceding N windows in
+addition to a 50% increase. If no amount is spent in the preceding windows,
+the maximum allowed amount equals a bootstrap amount in addition to a 50%
+increase.</blockquote>
+
+For mainnet, the expenditure policy window has a size of 6912 blocks and 6
+preceding windows are checked, for a total of 41472 blocks. A bootstrap amount
+of 16K DCR is used.
+
+A deeper overview of the issue can be read on the 
+[[https://blog.decred.org/2021/06/25/Treasury-Expenditure-Policy-Bug/|Treasury Expenditure Policy Bug]]
+blog post.
+
+==Specification==
+
+Change the maximum expenditure policy to the following:
+
+<blockquote>The sum of treasury spend amounts inside the most recent expenditure window
+cannot exceed the total amount received by the treasury in the same window, in
+addition to a 50% increase.</blockquote>
+
+==Rationale==
+
+Changing the expenditure policy to the one originally specified and approved by
+stakeholders in Politeia should not be controversial from a community discussion
+standpoint and solves the immediate issues facing the blocking of the funds by
+the treasury.
+
+The two policies are largely compatible at this point time, given the current
+average expenditure amount being withdrawn from the treasury, as well as the
+currently mined test transaction in mainnet being acceptable by both policies.
```

Comment:
This is really just a matter of style (plus a couple of typos), but I find this a little hard to parse.  How about the following?

```suggestion
The two policies are largely compatible at this point in time since the following relevant factors are valid under both policies:
- The current average expenditure amount being withdrawn from the treasury
- The existing <code>mainnet</code> test treasury spend transaction
```